contracts/v0.8/utils/Actor.sol

```diff
@@ -127,6 +127,21 @@ library Actor {
         return readRespData(data);
     }
 
+    /// @notice Readonly version of CallByID
+    /// @param target actor id (uint64) to interact with
+    /// @param method_num id of the method from the actor to call
+    /// @param codec how the request data passed as argument is encoded
+    /// @param raw_request encoded arguments to be passed in the call
+    /// @return payload (in bytes) with the actual response data (without codec or response code)
+    function callByIDReadOnly(CommonTypes.FilActorId target, uint256 method_num, uint64 codec, bytes memory raw_request) internal view returns (bytes memory) {
+        function(CommonTypes.FilActorId, uint256, uint64, bytes memory, uint256, bool) internal view returns (bytes memory) callFn;
+        function(CommonTypes.FilActorId, uint256, uint64, bytes memory, uint256, bool) internal returns (bytes memory) helper = callByID;
+        assembly {
+            callFn := helper
+        }
```

Comment:
This just hides it from the compiler, yes? Seems a little weird but I think that was acknowledged. Are there any trade-offs we are making here?